environments/qa/modules/localconfig/manifests/nodes.pp

```diff
@@ -1,10 +1,5 @@
-node 'qa0' {
-    $nodetype = 'qa'
+node 'dev0' {
```

Comment:
Now done in 60b7599 , though it wasn't quite that straightforward because the node name was previously hard-coded to dev0 in the provisioning script so that had to be changed too, which had a knock-on effect that the local env needed changing too.